lib/api/2.0/notification.js

```diff
@@ -12,6 +12,31 @@ var notificationPost = controller({success: 201}, function(req) {
     return notificationApiService.postNotification(message);
 });
 
+/**
+ * @api {get} /api/2.0/notification/:taskId/:steps
+ * @apiDescription notification get api, used to notify RackHD ipxe initrd downloading finished
+ *  :taskId: active taskId
+ *  :steps: task steps count
+ * @apiName notification-get
+ * @apiGroup notification
+ */
+var notificationIdGet = controller(function(req, res){
```

Comment:
Would suggest change function name from ```notificationIdGet``` to a simpler ```notificationGet``` or ```notifyProgressByTaskId```(since this function will only be used for sending progress notifications) or ```notifyProgressByGet```  or some other names that will reflect more precisely on it's function.